### ISSUE TITLE
Mount disk and start ssh on boot

### ISSUE THREAD

**Issue by afbjorklund** [boot2podman/boot2podman#1]
We need something like `podman-machine`, to start up the VM and to:
- download from releases and also cache the boot image (ISO)
- prepare a new disk image, that can be formatted (ext4) and mounted
- start ssh keygen and server, and prepare the user's ssh environment
- start podman varlink service, and set up the ssh keys to access it

**Comment by afbjorklund** [comment]
The mounting code can be adopted from: (though we **don't** need any swap)
https://github.com/boot2docker/boot2docker/blob/master/files/init.d/autoformat

Until the openssh.tcz is fixed (#3), the service can be loaded and started with:
``` sh
tce-load -i openssh
sudo cp /usr/local/etc/ssh/sshd_config.orig /usr/local/etc/ssh/sshd_config
sudo ssh-keygen -t ed25519 -N "" -f /usr/local/etc/ssh/ssh_host_ed25519_key
sudo /usr/local/etc/init.d/openssh start
```
The podman varlink service is already installed at boot and can be started with:
``` sh
sudo /usr/local/etc/init.d/podman start
```

We need one set of ssh keys for the `tc` user, and one set of keys for `root`.
The root keys will be used by podman/pypodman, for external varlink access.

**Comment by afbjorklund** [comment]
Fixed in v0.13
